default/python/AI/SaveGameManager/_decoder.py

```diff
@@ -0,0 +1,182 @@
+"""This module defines the decoding for the FreeOrion AI savegames.
+
+The decoder is subclassed from the standard library json decoder and
+uses its string parsing. However, the resulting objects will be interpreted
+differently according to the encoding used in FreeOrion AI savegames.
+
+The decoder will only load trusted classes as defined in _definitions.py,
+if an unknown/untrusted object is encountered, it will raise a InvalidSaveGameException.
+
+When classes are loaded, their __setstate__ method will be invoked if available or
+the __dict__ content will be set directly. It is the responsiblity of the trusted classes
+to provide a __setstate__ method to verify and possibly sanitize the content of the passed state.
+
+Example usage:
+    Importing this module directly is not advised.
```

Comment:
Importing `_*` is not advised by default, other methods are public.   Importing from __init__ instead of certain file is normal practice.  Is it special case here? Then describe it,  otherwise just remove this.